File: 0119-pascals-triangle-ii/0119-pascals-triangle-ii.java
class Solution {
    public List<Integer> getRow(int n) {
        if(n==0){
            List<Integer> l = new ArrayList<>();
            l.add(1);
            return l;
        }
        List<List<Integer>> l = new ArrayList<>();
        for(int i=0;i<=n;i++){
            ArrayList<Integer> l1 = new ArrayList<>();
            for(int j=0;j<=i;j++){
                if(j==0||j==i){
                    l1.add(1);
                }else{
                    List<Integer> x =l.get(i-1);
                    l1.add(x.get(j-1) + x.get(j));
                }
            }
            l.add(l1);
        }

        return l.get(n);
    }
}

File: 0119-pascals-triangle-ii/0119-pascals-triangle-ii.py
class Solution(object):
    def getRow(self, n):
        l = [1]
        for i in range(1,n+1):
            l.append(l[i-1]*i)
        
        x=l[len(l)-1]
        l1=[]
        for i in range(0,n+1):
            l1.append(x/(l[i]*l[n-i]))

        return l1
        